### PR TITLE
feat: add uncollected fees to dashboard

### PR DESCRIPTION
**Pull request by guerrap**
# Summary

Related to #43 

Add a data card to the dashboard for the uncollected fees across the different networks.

image

# How To Test

1.  Open the page `dashboard`
- [ ] Verify it contains the `Uncollected fees` data card

### CODE REVIEW

**Comment by 0xVenky** [comment]
LGTM! 🚀  Good job @guerrap :)

@wixzi you want to review this ? :)

File: src/contexts/Dashboard.js [hunk 1]
```diff
@@ -398,6 +422,41 @@ export const usePastMonthWalletsData = () => {
...
+    return {
+      [SupportedNetwork.MAINNET]: uncollectedFees.mainnetUSD,
+      [SupportedNetwork.ARBITRUM_ONE]: uncollectedFees.arbitrumOneUSD,
+      [SupportedNetwork.XDAI]: uncollectedFees.xDaiUSD,
+      total: uncollectedFees.totalUSD,
+    };
+  } catch (error) {
+    console.error(error);
+  }
+};
```
Comment (wixzi) [inline src/contexts/Dashboard.js:458]: may be move getUncollectedFees above the hook.

File: src/contexts/Dashboard.js [hunk 2]
```diff
@@ -138,6 +140,17 @@ function reducer(state, { type, payload }) {
       };
     }
 
+    case UPDATE_UNCOLLECTED_FEES_DATA: {
+      const { uncollectedFees } = payload;
+      return {
+        ...state,
+        feesData: {
+          ...state.feesData,
```
Comment (wixzi) [inline src/contexts/Dashboard.js:148]:
```suggestion
      
```
Are you sure you need old data?. Anyway the new objects will override the old ones.

**Comment by wixzi** [comment]
I have given you a patch in keybase. Please check

**Review by wixzi — CHANGES_REQUESTED** [review]
I have given you a patch in keybase. Please check

**Review by wixzi — APPROVED** [review]

**Pull request merged**